build.gradle

```diff
@@ -55,6 +54,14 @@ dependencies {
     api "com.exactpro.th2:grpc-service-generator:${serviceGeneratorVersion}"
 }
 
+sourceSets {
+    main {
+        resources {
+            srcDirs 'src/gen/main/services/java/resources'
```

Comment:
Have you checked the generation? I believe they are located in a different directory
See the example here
https://github.com/th2-net/th2-grpc-sim/blob/dev-version-5/build.gradle#L42

Reply:
My bad, thank you for recheck.